app/assets/javascripts/components/edit_profile_modal.es6.jsx

```diff
@@ -201,6 +224,14 @@ class EditProfileModal extends React.Component {
                   <div className="control__indicator"></div>
                 </label>
               </div>
+              <div className="input-field">
+                <label class>Profile Picture</label><br></br>
```

Comment:
This should be "className = ..." since we're in react, but we actually don't need this here!  (Also let's move the line break to a new line).
`<label htmlFor="profile-picture">Profile Picture</label>`
`<br/>`